src/picker_gui.cpp

```diff
@@ -490,6 +507,9 @@ void PickerWindow::OnInvalidateData(int data, bool gui_scope)
 		SetWidgetLoweredState(WID_PW_MODE_USED, HasBit(this->callbacks.mode, PFM_USED));
 		SetWidgetLoweredState(WID_PW_MODE_SAVED, HasBit(this->callbacks.mode, PFM_SAVED));
 	}
+
+	SetWidgetDisabledState(WID_PW_SHRINK, this->preview_height == PREVIEW_HEIGHT);
+	SetWidgetDisabledState(WID_PW_EXPAND, this->preview_height == PREVIEW_HEIGHT * 3);
```

Comment:
Constant?

`static const int MAX_PREVIEW_HEIGHT = PREVIEW_HEIGHT * 3;`

```suggestion
	SetWidgetDisabledState(WID_PW_EXPAND, this->preview_height == MAX_PREVIEW_HEIGHT);
```

Reply:
Forgot to use the new constant here. 😉